libopenage/gui/qml/IngameHud.qml

```diff
@@ -109,7 +109,7 @@ Item {
 					sourceComponent: resourceIndicator
 
 					onLoaded: {
-						item.amount = "13/42"
+						item.amount = root.actionMode.population
```

Comment:
`onLoaded` is executed only once. Wrap it in `Qt.binding(function() { return root.actionMode.population })` like for the resources.